README.md

```diff
@@ -38,14 +38,17 @@ New commands should go under commands.
 Ensure you are running go 1.7 or later.
 
 Install glide first [it's on GitHub](https://github.com/Masterminds/glide)
+```
+go get github.com/Masterminds/glide
```

Comment:
Don't I need to run `go install` to be able to use `rhm` from the command line?

I was under the impression that it was better practice to use `go install` instead of `go build`